web/packages/extension/4399_rules.json

```diff
@@ -0,0 +1,21 @@
+[
+  {
+    "id": 1,
+    "priority": 1,
+    "condition": {
+      "domains": ["4399.com"],
```

Comment:
`domains` is deprecated: https://developer.mozilla.org/en-US/docs/Mozilla/Add-ons/WebExtensions/API/declarativeNetRequest/RuleCondition#domains. This should probably use `initiatorDomains` instead, and it may be a good idea to further restrict it by `requestDomains`

Reply:
Updated to `initiatorDomains`. `requestDomains` may not work for this scenario because this site hosts swf files on many different CDNs. I have set `urlFilter` to filter out requests to swf files only.